docs/content/commands/npm-config.md

```diff
@@ -156,6 +158,13 @@ The command to run for `npm edit` and `npm config edit`.
 
 When passed to `npm config` this refers to which config file to use.
 
+When set to "global" mode, packages are installed into the `prefix` folder
+instead of the current working directory. See
+[folders](/configuring-npm/folders) for more on the differences in behavior.
+* packages are installed into the `{prefix}/lib/node_modules` folder,
+instead of the current working directory. * bin files are linked to
+`{prefix}/bin` * man pages are linked to `{prefix}/share/man`
```

Comment:
```suggestion
* packages are installed into the `{prefix}/lib/node_modules` folder, instead of the current working directory.
* bin files are linked to `{prefix}/bin`
* man pages are linked to `{prefix}/share/man`
```

Reply:
arguable that the first line here should be trimmed, but the main thing i wanted to correct is the missing line breaks